src/db/types.ts

```diff
@@ -18,23 +18,31 @@ export interface Rating {
 	_id?: ObjectId;
 	userID: ObjectId;
 	placeID: GooglePlace["place_id"];
-	braille: number | null;
-	fontReadability: number | null;
-	staffHelpfulness: number | null;
-	navigability: number | null;
 	guideDogFriendly: number | null;
+	isMenuAccessible: 0 | 1 | null; // null bc they may not be able to give a yes/no answer
```

Comment:
how? wouldn't this always be a yes or no?

Reply:
imo the frontend for this should be something like "was there something like a screen-reader compatible menu or braille menu so that you could read it and order yourself?"

Reply:
Not necessarily, we shouldn't assume that a user will have the answer for everything. In theory, yes they should have an answer, but we don't know that for sure

Reply:
wasn't the one who made this change, but I imagined that the entire rating would not be necessary to fill out in order to submit it - users could submit as much of the rating as they would like

Reply:
I personally think not giving the option of 'Don't know' or whatever we choose that 'null' option to be in the UI is bad design bc then we corner the user into making a decision even if they don't have an answer.

Reply:
we could also have some instructions that tell the user that they don't need to select a response for all ratings/yes/nos

Reply:
Agree @carpenterd777 either way, we need the null to cover that case

Reply:
ok, on the frontend side i STRONGLY don't think we should have an option that says "Don't know" because i think it makes it much more likely people will submit less data, and we really need that data for this app to be useful.

However, i think having some optional fields is fine, just not a button that might tempt people to just choose "Don't know" so null is fine for now

Reply:
@aej11a I agree that there should not be a "Don't know" option

Reply:
Yeah I agree with that too, as long as we have the option for a user to not give an answer, I think we're good

Reply:
This PR is only backend, so we can decide on the layout of the frontend in a later PR